tools/latency-monitor/main.go

```diff
@@ -0,0 +1,237 @@
+package main
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"math"
+	"os"
+	"os/signal"
+	"sync"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/celestiaorg/go-square/v2/share"
+
+	"github.com/celestiaorg/celestia-app/v4/app"
+	"github.com/celestiaorg/celestia-app/v4/app/encoding"
+	"github.com/celestiaorg/celestia-app/v4/pkg/user"
+)
+
+const (
+	defaultEndpoint     = "localhost:9090"
+	defaultKeyringDir   = "~/.celestia-app"
+	defaultSubmitRate   = 1.0    // KB per second
+	defaultBlobSize     = 1      // bytes
```

Comment:
I think the units here are wrong because later on this is used like so

```
				blobData := make([]byte, blobSize*1024)
```

```suggestion
	defaultBlobSize     = 1      // kibibytes
```